myapp.rb

```diff
@@ -6,3 +6,9 @@
   content_type :json
   Shop.all.to_json
 end
+
+delete '/v1/shops/:id/delete' do
+  shop = Shop.find(params[:id])
+  shop.delete
+  204
```

Comment:
204 should be an http response code, not a body response. look `Setting Body, Status Code and Headers` here: http://sinatrarb.com/intro.html